streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignorTest.java

```diff
@@ -104,6 +106,12 @@
     private final MockClientSupplier mockClientSupplier = new MockClientSupplier();
     private final TopologyBuilder builder = new TopologyBuilder();
     private final StreamsConfig config = new StreamsConfig(configProps());
+    private final StreamThread mockStreamThread = new StreamThread(builder, config,
```

Comment:
There are some tests using `onAssign` which were getting NPEs.  Since in practice the embedded `streamThread` should never be null, I opted to update the test.